SMBPositionAnalytics.md

```diff
@@ -0,0 +1,151 @@
+SMB Position Analytics Service
+==============================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Requests](#requests)
+    - [Carotene With Location Request](#carotene-with-location-request)
+    - [Job Request](#job-request)
+- [Responses](#responses)
+    - [Carotene Response Structure](#carotene-response-structure)
+    - [Job Response Structure](#job-response)
+- [Versioning](#versioning)
+
+## Summary
+
+The Small Medium Business(SMB) Position Analytics service provides hiring data pertaining to job titles and currently open positions available on CareeBuilder job boards. The service has two endpoints:
+
+https://api.careerbuilder.com/core/positionanalytics/carotene
+
+https://api.careerbuilder.com/core/positionanalytics/job
+
+Requests can be sent in the form of HTTP GET or POST (form or JSON)
+
+ This service requires Careerbuilder Oauth 2.0 client credentials. For more information please see our ReadMe [here](/Readme.md#access).
```

Comment:
beef comment: can we just put the link in the sentence? and can we call it 'documentation' instead of 'ReadMe'?

For more information please see the authorization [documentation](www.example.org).

Reply:
Casing on careerbuilder is inconsistent with your previous use.